src/main/java/net/fabricmc/loom/configuration/CompileConfiguration.java

```diff
@@ -83,6 +84,12 @@ public void run() {
 			javadoc.setClasspath(main.getOutput().plus(main.getCompileClasspath()));
 		});
 
+		// Make all outputs to be reproducible
+		getTasks().withType(AbstractArchiveTask.class).configureEach(abstractArchiveTask -> {
```

Comment:
Since `withType` is a live collection and `configureEach` is lazy, does this affect user-defined tasks? If it does, I'm wondering whether it'd be better to just set defaults for AbstractRemapJarTasks.

Reply:
That might be better, I had hoped to be able to set the convention on these.